src/lib/modules/guilds/components/Grid/type.ts

```diff
@@ -0,0 +1,9 @@
+export enum GridFlow {
+    ROW  = 'grid-flow-row',
+    COLUMN  = 'grid-flow-col'
+}
+
+export enum GridType  {
+    ROW = 'grid-row-',
+    COLUMN = 'grid-cols-'
```

Comment:
```suggestion
    COLUMN = 'grid-cols-'
```
Hello @arielrecto this will not work well with tailwind